packages/typespec-ts/test/modularUnit/scenarios/models/nestedEnum/notFlatten/experimentalExtensibleEnumsUndefined.md

```diff
@@ -40,10 +40,10 @@ withRawContent: true
 Model generated.
 
 ```ts models
-/** Alias for Foo */
-export type Foo = "bar" | Baz | string;
+/** Alias for _Foo */
+export type _Foo = "bar" | Baz | string;
```

Comment:
@tadelesh in this case, the Foo is defined as 
```
union Foo {
  "bar",
  Baz,
  string,
}
```
but the type we get from tcgc isGeneratedName is true 
<img width="750" alt="image" src="https://github.com/user-attachments/assets/344d650c-03a7-44ae-8a49-4c6fedca00a5" />